docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #148.        excl_zone_st = query_idx - excl_zone
</pre><p>It's not clear what <code>st</code> stands for (I think it's <code>start</code> but it's not obvious)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yes! I should have used the full word "start". I will fix it.</p>